in_toto/model.go

```diff
@@ -61,6 +66,9 @@ var ErrNoPublicKey = errors.New("the given key is not a public key")
 // for example: curve size = "521" and scheme = "ecdsa-sha2-nistp224"
 var ErrCurveSizeSchemeMismatch = errors.New("the scheme does not match the curve size")
 
+// ErrInvalidPayloadType indicates that the envelope used an unkown payload type
+var ErrInvalidPayloadType = fmt.Errorf("unknown payload type")
```

Comment:
Is there a reason for using fmt.Errorf instead of errors.New here? You don't seem to need the formatting + if we use errors.New() we can still wrap additional information around it via: fmt.Errorf("%w: %s", ErrInvalidPayloadType, someUsefulInformation).

See also: https://blog.golang.org/go1.13-errors

Reply:
No, just an old habit. Errors created via `fmt.Errorf` can be wrapped too, as if no error is provided as an argument `fmt.Errorf` is just a pass-through to `errors.New()`. But as we don't need the formatting here I'll switch 👍